cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -218,7 +218,7 @@ public void execute() {
                 //// check if the given file exists.
                 if (Files.notExists(sourcePath)) {
                     CommandUtil.printError(this.errStream,
-                            "'" + sourcePath + "' ballerina file does not exist.",
+                            "The Ballerina file '" + sourcePath + "' does not exist.",
```

Comment:
Or can simply say: `File not found: <path_to_the_file>`

Reply:
Fixed as follows:

`"file not found: '" + sourcePath,`